lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'dish'
+require_relative 'twilio'
+
+class Order
+  attr_reader :dishes, :order
+
+  def initialize
+    @dishes = Dish.new
```

Comment:
So are we dealing with plural dishes here, as the name of the instance variable would suggest, or a singular dish, as the classname would suggest?